### PR TITLE
SL-20226 Object Profile Floater Stays As Inactive Transparency When Active

### PR DESCRIPTION
**Pull request by LLGuru**

### CODE REVIEW

**Review by akleshchev — APPROVED** [review]
Fix looks good in scope of sidepanel container, but DRTVWR-567 probably is a better place for this as it havily affect object profiles.

File: indra/newview/llfloatersidepanelcontainer.cpp [hunk 1]
```diff
@@ -99,6 +99,10 @@ LLPanel* LLFloaterSidePanelContainer::openChildPanel(const std::string& panel_na
 	{
 	openFloater();
 	}
+	else if (!hasFocus())
+	{
+		setFocus(TRUE);
```
Comment (akleshchev) [inline indra/newview/llfloatersidepanelcontainer.cpp:104]: This probably should do a bringtofront, not just setfocus

Comment (LLGuru) [reply indra/newview/llfloatersidepanelcontainer.cpp:104]: It works as expected with  this change

**Review by akleshchev — APPROVED** [review]

**Comment by AtlasLinden** [comment]
Archived issue: https://github.com/secondlife/jira-archive-internal/issues/70573

**Pull request merged**